app/_src/gateway/production/deployment-topologies/index.md

```diff
@@ -49,7 +49,7 @@ In [traditional mode](/gateway/{{page.kong_version}}/production/deployment-topol
 
 Running {{ site.base_gateway }} in traditional mode is the simplest way to get started with Kong, and it is the only deployment topology that supports plugins that require a database, like rate-limiting with the cluster strategy, or OAuth2. However, there are some downsides too.
 
-When running in traditional mode, every {{ site.base_gateway }} node runs as both a Control Plane (CP) and Data Plane (DP). This means that if **any** of your nodes are compromised, the entire running gateway configuration is compromised. In contrast, hybrid mode (shown below) has distinct CP and DP nodes reducing the attack surface.
+When running in traditional mode, every {{ site.base_gateway }} node runs as both a Control Plane (CP) and Data Plane (DP). This means that if **any** of your nodes are compromised, the entire running gateway configuration is compromised. In contrast, hybrid mode (shown above) has distinct CP and DP nodes reducing the attack surface.
```

Comment:
```suggestion
When running in traditional mode, every {{ site.base_gateway }} node runs as both a Control Plane (CP) and Data Plane (DP). This means that if **any** of your nodes are compromised, the entire running gateway configuration is compromised. In contrast, [hybrid mode](#hybrid-mode) has distinct CP and DP nodes reducing the attack surface.
```
This "shown above/below" is a remnant from a time when there was an image on the page; that image is now on the hybrid mode doc itself ([here](https://deploy-preview-5514--kongdocs.netlify.app/gateway/latest/production/deployment-topologies/hybrid-mode/)). I think it might make more sense to take that out entirely and link to the section instead.

Reply:
Did you want to edit as desired or suggesting I replace mention and make "hybrid mode" simply a link to the doc URL you shared as "here" in your comment?

Reply:
As a reader, which link would make more sense to you here? The one I suggested links to the short description, then you can choose to read more about hybrid mode if you need to. Or, we can link to `/gateway/latest/production/deployment-topologies/hybrid-mode/`, and skip the interim step.

Reply:
I like the link you mentioned "here" above because it has a diagram and clear description if they chose to follow. Given they already read the sections above, however, they should know what it is so that would just be an optional deep dive.